pytoda/datasets/protein_sequence_dataset.py

```diff
@@ -153,6 +172,40 @@ def __init__(
                 Sequences. Defaults to False.
             randomize (bool): perform a true randomization of Protein tokens.
                 Defaults to False.
+            sequence_augment (Dict): a dictionary to specify additional sequence
+                augmentation. Defaults to {}. Items can be:
+                    `alignment_path`: A path (str) to a `.smi` (or `.tsv`) file
+                        with alignment information that specifies which residues of the
+                        sequence are to be used. E.g., to extract active site sequences
+                        from full proteins. Do not use a header in the file.
+                        1. column has to be the full protein sequence (use upper
+                            case only for residues to be used). E.g., ggABCggDEFgg
+                        2. column has to be the condensed sequence (ABCDEF).
+                        3. column has to be the protein identifier.
+                        NOTE: Such a file is *necessary* to apply all augmentation types
+                        specified in this dictionary (`sequence_augment`).
+                        NOTE: Unless specified, this defaults to
+                        `kinase_activesite_alignment.smi`, a file in
+                        `pytoda.proteins.metadata` that can ONLY be used to
+                        extract active site sequences of human kinases (based on the
+                        active site definition in Sheridan et al. (2009, JCIM) and
+                        Martin & Mukherjee (2012, JCIM).
+                    `discard_lowercase`: A (bool) specifying whether all lowercase
+                        characters (residues) in the sequence should be discarded.
+                        NOTE: This defaults to True.
+                    `flip_substrings`: A probability (float) to flip each substring
+                        (e.g., an active site substring that lies contiguously in the
+                        original sequence). Defaults to 0.0, i.e., no flipping.
+                        E.g., ABCDEF could become CBADEF or CBAFED or ABCFED if the
+                        original sequence is ggABCggDEFgg.
+                    `swap_substrings`: A probability (float) to swap neighboring
+                        substrings. Defaults to 0.0, i.e., no swapping.
+                        E.g., ABCDEF could become DEFABC if the original sequence is
+                        ggABCggDEFgg.
+                    `noise`: A 2-Tuple of (float, float) that specifies the probability
+                        for a random, single-residue mutation inside and outside the
+                        relevent part. Defaults to (0.0, 0.0), i.e., no noise.
+                        E.g., with (0.0, 0.5),  ggABCggDEFgg could become hgABCgbDEFgg.
```

Comment:
maybe it will be good to add few examples of complete values of sequence_augment

Reply:
This can be seen from the unittests if needed I think